syncer/data_fetch.go

```diff
@@ -326,6 +326,12 @@ func (d *DataFetch) GetEpochATXs(ctx context.Context, epoch types.EpochID) error
 		log.Int("missing", len(missing)),
 	)
 	if len(missing) > 0 {
+		d.logger.WithContext(ctx).With().Info("fetching missing epoch atxs",
+			epoch,
+			log.Stringer("peer", peer),
+			log.Int("total", len(ed.AtxIDs)),
```

Comment:
afaik, this total is not the absolute total, so we need to be careful here.

maybe @dshulyak can point to more, but I remember that I had a similar idea some time ago, and that's how I remembered.

Reply:
i think this is from epoch info response, so in this case this is how many atxs are available from that peer. definitely not absolute, and not even how many are available locally 

> ed, err := d.fetcher.PeerEpochInfo(ctx, peer, epoch)